readme.md

```diff
@@ -89,23 +94,33 @@ var clown = {
 Access the value `"Sue"` out of the `names` array:
 
 ```js
-answer goes here
+
+names(2)
+
+
 ```
 
 ## Question 9
 
 Write a for loop such that you say hello(using `console.log`) to each name in the `names` array
 
 ```js
-answer goes here
+
+for (names = 0, 1, 2, 3) {
+	console.log ("Hello")
+	}
+	
+
 ```
 
 ## Question 10
 
 Access the value `"Alfred"` out of the `clown` object
 
 ```js
-answer goes here
+
+clown = minions(1)
+
 ```
 
```

Comment:
Always test your code! I can see evidence of an informed thought process as you thought about the last 4 answers, though.